rules/linux/execution_cpulimit_binary.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2022/03/17"
+maturity = "production"
+updated_date = "2022/03/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary cpulimit abuse to break out from restricted environments by spawning an interactive system
+shell.The cpulimit utility is used to restrict the CPU usage of a process in cases of CPU or system load exceeding the
+defined threshold and the activity of spawing a shell is not a standard use of this binary by a user or system
+administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their
+access.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via cpulimit Shell evasion"
+references = ["https://gtfobins.github.io/gtfobins/cpulimit/"]
+risk_score = 47
+rule_id = "0968cfbd-40f0-4b1c-b7b1-a60736c7b241"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id, process.pid with maxspan=1m
```

Comment:
why was it chosen to do this as a sequence vs using `process.parent.*` as a flat rule?

Reply:
This cannot be a flat rule, as the shell fork happens once the parent command has completed execution and a new fork, hence the sequence requirement

Reply:
Also, the sequence provides better context for the analyst. We can provide this event as a sequence providing the analyst definitive proof and assurance that the attempt to spawn an interactive shell was successful rather than them having to dig and validate that for themselves.